justfile

```diff
@@ -149,7 +149,7 @@ test *flags:
 
 # Build linkerd-await
 build:
-    {{ _cargo }} build --frozen --target={{ _cargo_target }} \
```

Comment:
I don't think that this is enough to safely distribute a binary. As I search around, I find a bunch of information suggesting that there are problems statically linking against glibc. However, we have prior art for linking against musl:

https://github.com/linkerd/linkerd2/blob/b16f3f0825b8639b889501cb24583cb9d3d22002/policy-controller/Dockerfile#L13-L22

https://github.com/olix0r/hokay/blob/ad64b990bf70cebbe939d5a6e81d16ac837ce443/.github/workflows/release.yml#L56-L150

This requires some CI surgery, but I can port one of these over.

Before doing that, though, I want to confirm that the original problem we're solving can't be more easily solved by rebuilding the binaries against a newer version of glibc.

Reply:
Ah. It looks like distributing binaries statically linked against GNU libc (LGPLv2.1) may have licensing implications, the extent of which I do not immediately understand.